website/guides/running-terraform-in-automation.html.md

```diff
@@ -0,0 +1,279 @@
+---
+layout: "guides"
+page_title: "Running Terraform in Automation - Guides"
+sidebar_current: "guides-running-terraform-in-automation"
+description: |-
+  Terraform can, with some caveats, be run in automated processes such as
+  continuous delivery pipelines. Ths guide describes some techniques for
+  doing so and some gotchas to watch out for.
+---
+
+# Running Terraform in Automation
+
+~> **This is an advanced guide!** When getting started with Terraform, it's
+recommended to use it locally from the command line. Automation can become
+valuable once Terraform is being used regularly in production, or by a larger
+team, but this guide assumes familiarity with the the normal, local CLI
+workflow.
+
+For teams that use Terraform as a key part of a change management and
+deployment pipeline, it can be desirable to orchestrate Terraform runs in some
+sort of automation in order to ensure consistency between runs, and provide
+other interesting features such as integration with version control hooks.
+
+Automation of Terraform can come in various forms, and to varying degrees.
+Some teams continue to run Terraform locally but use _wrapper scripts_ to
+prepare a consistent working directory for Terraform to run in, while other
+teams run Terraform entirely within an orchestration tool such as Jenkins.
+
+This guide covers some things that should be considered when implementing
+such automation, both to ensure safe operation of Terraform and to accommodate
+some current limitations in Terraform's workflow that require careful
+attention in automation.
+
+The guide assumes that Terraform will be running in an _non-interactive_
+environment, where it is not possible to prompt for input at the terminal.
+This is not necessarily true for wrapper scripts, but is often true when
+running in orchestration tools.
+
+This is a general guide, giving an overview of things to consider when
+implementing orchestration of Terraform. Due to its general nature, it is not
+possible to go into specifics about any particular tools, though other
+tool-specific guides may be produced later if best practices emerge around
+such a tool.
+
+## Automated Workflow Overview
+
+When running Terraform in automation, the focus is usually on the core
+plan/apply cycle. The main path, then, is the broadly same as for CLI
+usage:
+
+1. Initialize the Terraform working directory.
+2. Terraform produces a plan for changing resources to match the current
+configuration.
```

Comment:
I don't think adding that extra word adds anything here. Terraform doesn't see configuration _changes_, it sees differences between state and configuration, which may be the result either of changes to the configuration or of drift.